service/src/main/java/greencity/service/notification/NotificationServiceImpl.java

```diff
@@ -787,6 +787,16 @@ public void deleteNotification(Long notificationId, String userUuid) {
         userNotificationRepository.markAsDeletedUserNotificationByIdAndUserId(notificationId, userId);
     }
 
+    @Override
+    public void notifyUnpaidOrderPermanently(Order order, Long amountToPay) {
```

Comment:
based on the java doc this method should be run `by schedule task, which runs every day.`
But I see that it is called in `saveFullOrderToDB`